### DISCUSSION TITLE
How to force Reforge to not use my storage when switching checkpoint?

### DISCUSSION THREAD

**Discussion by koopa990** [Panchovix/stable-diffusion-webui-reForge#93] [Q&A]
Is there settings in the UI or?

**Comment by Panchovix** [comment]
What do you mean by using storage? You mean having everything all the time on GPU? You can use --always-gpu and --pin-shared-memory

**Reply by koopa990** [reply]
> What do you mean by using storage? You mean having everything all the time on GPU? You can use --always-gpu and --pin-shared-memory

whenever I switch to different checkpoint models, Reforge temporarily consumes my storage device. On regular Forge, it NEVER did that.

**Reply by Panchovix** [reply]
Not sure why that would be happening in this case, specifically on main branch since it's the same backend as OG Forge (before gradio 4 update)

Without any logs it's hard to replicate.

You are welcome to use OG Forge before gradio 4 update if that fixes your issues!
